components/NaveVerticalCarousel.vue

```diff
@@ -1,57 +1,80 @@
 <template>
-  <div class="flex items-center nave-vertical-carousel w-full">
-    <div class="flex flex-col h-70 justify-between nave-vertical-carousel__slider-items-wrapper relative w-8">
-      <div v-for="n in 7" :key="`number-${n}`" class="flex font-bold h-8 items-center justify-center nave-vertical-carousel__slide-number text-black w-full z-2" :class="activateNumber(n - 1)">{{ n }}</div>
+  <div class="flex flex-col-reverse items-center lg:flex-row lg:order-1 nave-vertical-carousel w-full">
+    <div class="flex flex-row h-auto justify-between lg:flex-col lg:h-70 lg:mt-0 lg:w-8 mt-20 nave-vertical-carousel__slider-items-wrapper relative sm:w-70 w-64">
+      <div v-for="n in phrasesLength" :key="n" class="cursor-pointer flex font-bold h-6 items-center justify-center lg:w-full nave-vertical-carousel__slide-number sm:h-8 sm:w-8 text-black w-6 z-2" :class="activateNumber(n - 1)" @click="changeActiveIndex(n)">{{ n }}</div>
```

Comment:
`n` coloque um nome que faça mais sentido, se for algo muito generico chame de `item